code/modules/client/preference/preferences.dm

```diff
@@ -2230,8 +2189,35 @@ GLOB.glassesblock
 		else if(firstspace == name_length)
 			real_name += "[pick(GLOB.last_names)]"
 
-	/* Disabilities, Oh My Freakin' Heck, Y'all are Killing Voxxy */
-	character.add_language(language)			//Add Secondary Language to list of languages our character can speak
+	character.add_language(language) //Add Secondary Language to list of languages our character can speak
+
+	/*
+	Disabilities,
+	Oh My Freakin' Heck,
+	Y'all are Killing Voxxy
+	*/
+
+	var/list/dblock_list = new(GLOB.glassesblock, GLOB.colourblindblock, GLOB.blindblock, GLOB.deafblock, GLOB.muteblock, GLOB.fatblock, GLOB.swedeblock, GLOB.chavblock, GLOB.lispblock, GLOB.clumsyblock)
```

Comment:
THIS LIST HAS TO GO HERE, OMG, IT CAN'T GO ANYWHERE ELSE /s

but really, it has to go here, and that's okay, because this is the only place it is used.
why? because if it used before this point, the data (the block values) aren't ready to be pulled. GLOB.<block> values aren't set until `setupgame.dm` is done running and that dm is called _after_ `preferences.dm` is called.